R/dependencies.R

```diff
@@ -80,12 +80,19 @@ snapshotDependencies <- function(appDir, implicit_dependencies=c()) {
   } else {
     biocPackages = c()
   }
+  repo.packages <- available.packages(contriburl = contrib.url(repos), type = "source")
```

Comment:
`contrib.url()` uses the `pkgType` option, which differs by default on the different platforms. On macOS + Windows, it defaults to `"both"` effectively meaning "search both binary and source repositories". I'm guessing you always want to install from source so you might want to enforce `pkgType = "source"`.

Reply:
(It looks like this still returns a source repository when `pkgType = "both"` on my macOS machine + R 3.5.1 but better to be safe and explicit I think)